tests/python/gaia-ui-tests/gaiatest/tests/functional/homescreen/test_homescreen_column_layout.py

```diff
@@ -30,7 +30,9 @@ def test_homescreen_column_layout(self):
         settings.launch()
 
         homescreen_settings = settings.open_homescreen()
-        homescreen_settings.select_icon_layout('Four Columns')
+        homescreen_settings.select_change_icon_layout()
```

Comment:
In this test, I'd use only one function to `select` and `pick`. It'll make the test a bit easier to read I don't see a case (in our current test suite) where we tap on the button and cancel it. So we'd have:
```py
homescreen_settings.change_icon_layout('Four Columns')
```
```py
def change_icon_layout(self, layout_text):
    self._select_change_icon_layout()
    self._pick_icon_layout(text)
```
`_select_change_icon_layout()` and `_pick_icon_layout()` are protected method, as we don't use them outside anymore